.github/actions/install-prek/action.yml

```
@@ -16,38 +16,33 @@
 # under the License.
 #
 ---
-name: 'Install pre-commit'
-description: 'Installs pre-commit and related packages'
+name: 'Install prek'
+description: 'Installs prek and related packages'
 inputs:
   python-version:
     description: 'Python version to use'
     default: "3.10"
   uv-version:
     description: 'uv version to use'
     default: "0.8.9"  # Keep this comment to allow automatic replacement of uv version
-  pre-commit-version:
-    description: 'pre-commit version to use'
-    default: "4.3.0"  # Keep this comment to allow automatic replacement of pre-commit version
-  pre-commit-uv-version:
-    description: 'pre-commit-uv version to use'
-    default: "4.1.4"  # Keep this comment to allow automatic replacement of pre-commit-uv version
-  skip-pre-commits:
-    description: "Skip some pre-commits from installation"
+  prek-version:
+    description: 'prek version to use'
+    default: "0.0.25"  # Keep this comment to allow automatic replacement of prek version
+  skip-prek-hooks:
+    description: "Skip some prek hooks from installation"
     default: ""
 runs:
   using: "composite"
   steps:
-    - name: Install pre-commit, uv, and pre-commit-uv
+    - name: Install prek and uv
       shell: bash
       env:
         UV_VERSION: ${{inputs.uv-version}}
-        PRE_COMMIT_VERSION: ${{inputs.pre-commit-version}}
-        PRE_COMMIT_UV_VERSION: ${{inputs.pre-commit-uv-version}}
-        SKIP: ${{ inputs.skip-pre-commits }}
+        PREK_VERSION: ${{inputs.prek-version}}
+        SKIP: ${{ inputs.skip-prek-hooks }}
       run: |
         pip install uv==${UV_VERSION} || true
-        uv tool install pre-commit==${PRE_COMMIT_VERSION} --with uv==${UV_VERSION} \
-        --with pre-commit-uv==${PRE_COMMIT_UV_VERSION}
+        uv tool install prek==${PREK_VERSION} --with uv==${UV_VERSION}
```

```suggestion
        uv tool install prek==${PREK_VERSION}
```

I think it's not necessary to install prek `--with uv` since uv is already installed and prek will reuse that.

---

the idea is that we want to make sure to use the specific version of UV -> UV changes fast and occasionally things are broken, so we want to "freeze" the `uv` everywhere.